### DISCUSSION TITLE
How to programatically enable "realtime" for tables on deployment?

### DISCUSSION THREAD

**Discussion by WesleyYue** [supabase/supabase#11573] [Questions]
We deploy migrations to our environments via migration scripts created by `supabase db diff`, and push changes base on the migration scripts in our repo via CI. However, these autogenerated migration scripts don't hold the information of whether a table is marked as "Enable realtime" (via the studio UI) or not. Is there a way to do this programmatically so we can check it into our repo and CI process just like our SQL migration scripts?

**Comment by arshamg** [comment]
Also very interested in this 🙏

**Comment by mjarkk** [comment]
You can try add the following to a migration.

```sql
alter publication supabase_realtime add table public.YOUR_TABLE_NAME_HERE;
```
